wasm/binary/names_test.go

```diff
@@ -103,34 +109,6 @@ func TestEncodeNameSectionData(t *testing.T) {
 				0x01, 0x01, 'r', // index 1, size of "r", "r"
 			},
 		},
-		{
```

Comment:
we decode in order and don't pass anything into a map that would lose the order, so we don't need this anymore